exonum/src/blockchain/mod.rs

```diff
@@ -273,15 +268,13 @@ impl BlockchainMut {
     /// with the hash of the resulting block.
     pub fn create_patch(
         &self,
-        proposer_id: ValidatorId,
+        proposer_id: Option<ValidatorId>,
```

Comment:
Well, when I mentioned the optional storing of the `ValidatorId`, I didn't mean that it should become optional in any other context. Currently we use it, and do it for a reason, so I think that on the method signatures level everything should be kept the same.